### PR TITLE
Xcode 7.3 / Swift 2.2 

### PR DESCRIPTION
**Pull request by Dwar3xwar**

### CODE REVIEW

**Comment by bphenriques** [comment]
Thank you!

File: malaria-ios/ViewControllers/DailyStats/DailyStatsTableViewController.swift [hunk 1]
```diff
@@ -75,7 +75,7 @@ extension DailyStatsTableViewController{
     cell.backgroundColor = cell.backgroundColor
     
     // Social Integration with Stat Label
-    let tapGesture = UITapGestureRecognizer(target: self, action: "didPressInfo:")
+    let tapGesture = UITapGestureRecognizer(target: self, action: #selector(DailyStatsTableViewController.didPressInfo(_:)))
```
Comment (bphenriques) [inline malaria-ios/ViewControllers/DailyStats/DailyStatsTableViewController.swift]: Why not just didPressInfo?

**Comment by bphenriques** [comment]
Selectors without strings and with auto completion is much better :+1:

File: malaria-ios/ViewControllers/DailyStats/DailyStatsTableViewController.swift (hunk 1 above)
Comment (Dwar3xwar) [inline malaria-ios/ViewControllers/DailyStats/DailyStatsTableViewController.swift]: I mindlessly changed this one with the complier warning :laughing:

**Comment by Dwar3xwar** [comment]
Agreed, Xcode is getting better with each release :+1:

**Comment by bphenriques** [comment]
Thank you :)

**Pull request merged**